examples/mux_audio_and_video.exs

```diff
@@ -0,0 +1,35 @@
+Mix.install([
+  {:membrane_mpegts_plugin, path: "../"},
+  :membrane_file_plugin
```

Comment:
please specify the version of file plugin, so possible future breaking changes there won't break this example

Reply:
done